### PR TITLE
Extension: Getting Started

### PR DESCRIPTION
**Pull request by DavidSouther**
1. Extension generate
2. Extension edit
3. Extension messages
4. Extension "Currently generating" status count
    * Hides a couple seconds after last item finished.

### CODE REVIEW

File: extension/src/settings.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ function aillyLogFormatter<
   delete debug.message;
   delete debug.prefix;
   delete debug.level;
-  delete debug.source;
+  // delete debug.source;
```
Comment (cpyle0819) [inline extension/src/settings.ts:33]: Delete?

Comment (DavidSouther) [reply extension/src/settings.ts]: Done

File: extension/src/status_manager.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    this.outstanding += count;
+    this.updateStatusBarItem();
+  }
+  finishOutstanding() {
+    this.outstanding -= 1;
+    this.updateStatusBarItem();
+  }
+
+  updateStatusBarItem() {
+    this.statusBarItem.text = `Ailly: ${this.outstanding}`;
```
Comment (cpyle0819) [inline extension/src/status_manager.ts:47]: Is this useful? As a user I don't know how much I care about active threads. I _do_ care about my VSCode status bar real-estate.

**Review by cpyle0819 — APPROVED** [review]

File: extension/src/status_manager.ts (hunk 2 above)
Comment (DavidSouther) [reply extension/src/status_manager.ts]: Added a (maybe configurable in the future) delay to hide the status bar after getting to 0 outstanding items.

**Pull request merged**